reana_db/models.py

```diff
@@ -133,6 +137,25 @@ def get_user_disk_usage(self, block_size=None):
         except Exception:
             return -1
 
+    def get_user_cpu_usage(self):
+        """Aggregate user CPU usage."""
+        try:
+            from .database import Session
+
+            cpu_resources = [
+                str(res[0])
+                for res in Session.query(Resource.id_)
+                .filter_by(type_=ResourceType.cpu)
+                .all()
+            ]
+            return (
+                Session.query(func.sum(UserResource.quota_used))
+                .filter(UserResource.resource_id.in_(cpu_resources))
```

Comment:
Aren't we missing something like `UserResource.user_id == self.id_` to filter by user here?